eth/tracers/api.go

```diff
@@ -961,8 +961,16 @@ func (api *API) TraceCall(ctx context.Context, args ethapi.TransactionArgs, bloc
 		return nil, err
 	}
 
-	if api.backend.ChainConfig().IsOptimismPreBedrock(block.Number()) {
-		return nil, errors.New("l2geth does not have a debug_traceCall method")
+	if api.backend.ChainConfig().IsPreCel2(block.Time()) {
```

Comment:
Perhaps we can combine both conditionals.

Reply:
I don't see a simple way to do it since we have three different results:
* Return result from local node
* Return proxies result
* Return `rpc.ErrNoHistoricalFallback`

Reply:
If you combined them then you'd have to repeat one of the conditions though, right?